tap_typeform/streams.py

```diff
@@ -174,10 +174,10 @@ def sync_form(atx, form_id, start_date, end_date):
                     answer_value = answer[data_type]
 
                 answers_data_rows.append({
-                    "landing_id": row['landing_id'],
-                    "question_id": answer['field']['id'],
-                    "type": answer['field']['type'],
-                    "ref": answer['field']['ref'],
+                    "landing_id": row.get('field',{}).get('landing_id'),
```

Comment:
This doesn't need to `.get('field', {})`, it can just be `row.get('landing_id')`

Reply:
I reverted that line.